apollo-router/src/plugins/fleet_detector.rs

```diff
@@ -23,6 +23,7 @@ use crate::plugin::PluginPrivate;
 
 const REFRESH_INTERVAL: Duration = Duration::from_secs(60);
 const COMPUTE_DETECTOR_THRESHOLD: u16 = 24576;
+const OFFICIAL_HELM_CHART_VAR: &str = "APOLLO_ROUTER_OFFICIAL_HELM_CHART";
```

Comment:
does `OFFICIAL` distinguish between another env var that represents unofficial helm charts? wondering if it does any work in the name